src/sbl_filing_api/entities/repos/submission_repo.py

```diff
@@ -113,6 +123,28 @@ async def update_submission(submission: SubmissionDAO, incoming_session: AsyncSe
         raise
 
 
+async def update_submission_accepter(
+    session: AsyncSession,
+    submission_id: int,
+    accepter: str,
+    accepter_name,
+    accepter_email,
+) -> SubmissionAccepterDAO:
+    sub_accepter = await get_submission_accepter(session, submission_id=submission_id)
+    if sub_accepter:
+        sub_accepter.accepter = accepter
+        sub_accepter.accepter_name = accepter_name
+        sub_accepter.accepter_email = accepter_email
+    else:
+        sub_accepter = SubmissionAccepterDAO(
+            submission=submission_id,
+            accepter=accepter,
+            accepter_name=accepter_name,
+            accepter_email=accepter_email,
+        )
```

Comment:
The upsert does this.  You don't need to check here for an accepter.  Just create a DAO and send it to upsert, which will copy it and "merge".  The merge looks for an existing and if it's there, updates based on the new object.  If one doesn't exist, it creates a new one in the db.